test/controller.go

```diff
@@ -90,6 +92,7 @@ type Informers struct {
 	Condition        informersv1alpha1.ConditionInformer
 	Pod              coreinformers.PodInformer
 	ConfigMap        coreinformers.ConfigMapInformer
+	ServiceAccount   coreinformers.ServiceAccountInformer
```

Comment:
@vdemeester I am trying to understand this, why making this as part of informer? and creating fake service account informer?